unittests_that_fail_sporadically.txt

```diff
@@ -4,3 +4,11 @@ mongoOntimeintervalOperations.mongoGetContextElementResponses_ok (kz: Dec 17 201
 mongoOntimeintervalOperations.mongoUpdateCsubNewNotification_dbfail (kz: Dec 19 2014)
```

Comment:
Instead of a log-like list, maybe it is better to have the test ordered by number of times a sporadical fail happens, e.g.:

```
mongoUnsubscribeContext.x: 5 times
mongoUnsubscribeContext.y: 3 times
etc.
```

Thus, each time a new test fails, you add one to the right coner (and eventually reorder the lines if needed).